collect-settings.rst

```diff
@@ -77,9 +77,17 @@ Map settings are included on the :menuselection:`User Interface` settings page.
 Form Management Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: /img/collect-settings/form-management.*
+.. image:: /img/collect-settings/form-management.png
   :alt: Form Management settings
   :class: device-screen-vertical
+  :width: 49 %
+  
+.. image:: /img/collect-settings/form-management2.png
+  :alt: Form Management settings
+  :class: device-screen-vertical
+  :align: right
+  :width: 49 %
+
```

Comment:
Image directives do not need, and should not have, `align` or `width` parameters.

(Mixing display styling into content should be avoided whenever possible.)